contracts/src/token/erc721/extensions/burnable.rs

```diff
@@ -15,25 +15,24 @@ pub trait IErc721Burnable {
     /// The approval is cleared when the token is burned. Relies on the `_burn`
     /// mechanism.
     ///
+    /// # Requirements:
+    ///
+    /// * `token_id` must exist.
+    /// * The caller must own `token_id` or be an approved operator.
+    ///
     /// # Arguments
     ///
     /// * `value` - Amount to be burnt.
     ///
     /// # Errors
     ///
-    /// If token does not exist, then the error [`Error::NonexistentToken`] is
-    /// returned.
-    /// If the caller does not have the right to approve, then the error
-    /// [`Error::InsufficientApproval`] is returned.
-    ///
-    /// # Requirements:
-    ///
-    /// * `token_id` must exist.
-    /// * The caller must own `token_id` or be an approved operator.
+    /// * [`Error::NonexistentToken`] - If token does not exist.
+    /// * [`Error::InsufficientApproval`] - If the caller does not have the
+    ///   right to approve.
     ///
     /// # Events
     ///
-    /// Emits a [`super::super::Transfer`] event.
+    /// * [`super::super::Transfer`].
```

Comment:
```suggestion
    /// * [`erc721::Transfer`].
```
Import `erc721::{self, ...}` and use it instead of `super::super::` and [below](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/497/files#diff-63119dbcdb15383d1b4d07d39e0c4aa19dc9466da28692fad1d3eb122ca38bacL45)